bin/tls-include.sh

```diff
@@ -18,11 +18,31 @@ function verify_cert_manager {
   cert_manager_ok="true"
   if [ "$TLS_ENABLE" == "true" ]; then
```

Comment:
TLS is always required for logging regardless of the value of TLS_ENABLE is true or not.  Therefore, this function probably shouldn't skip processing if TLS_ENABLE is set to "false".  Or, alternatively, perhaps s we should set TLS_ENABLE = true in the deploy_logging_open.sh script?  Thoughts?

Reply:
My take: Override and always set to true in the logging script.

Reply:
Done